components/schema-migrator/migrations/director/202003041248_allow_tenant_deletion.up.sql

```diff
@@ -0,0 +1,46 @@
+ALTER TABLE api_definitions
```

Comment:
you can check if such syntax will be easier to read:
```
ALTER TABLE public.scores
DROP CONSTRAINT scores_gid_fkey,
ADD CONSTRAINT scores_gid_fkey
   FOREIGN KEY (gid)
   REFERENCES games(gid)
   ON DELETE CASCADE;
```

https://stackoverflow.com/questions/10356484/how-to-add-on-delete-cascade-constraints